scripts/hostmap-crtsh.nse

```diff
@@ -63,20 +40,16 @@ local target = require "target"
 local table = require "table"
 local tableaux = require "tableaux"
 
--- Different from stdnse.get_hostname
--- this function returns nil if the host is only known by IP address
+-- Return nil if host only has IP address
 local function get_hostname (host)
   return host.targetname or (host.name ~= '' and host.name) or nil
 end
 
--- Run on any target that has a name
 hostrule = get_hostname
 
+-- validate hostname structure
 local function is_valid_hostname (name)
   local labels = stringaux.strsplit("%.", name)
-  -- DNS name cannot be longer than 253
-  -- do not accept TLDs; at least second-level domain required
-  -- TLD cannot be all digits
```

Comment:
Leave the original comment. Focus your PR on the core enhancement, avoiding incidental changes.